tests/units/RedisMockFactory.php

```diff
@@ -170,6 +171,55 @@ public function testOrphanizeConstruct() {
                     ->notExists();
     }
 
+    /**
+     * Build mock by using 'orphanizeConstruct' parameter
+     *
+     * @php >= 7.1
+     * @return void
+     */
+    public function testOrphanizeConstructAfterPhp7Dot1() {
+        $factory = new Factory();
+        $this->assert
+            ->exception(function() use ($factory) {
+                $factory->getAdapter('M6Web\Component\RedisMock\tests\units\RedisWithNativeConstructor', false, false);
+            })
+            ->isInstanceOf(\ArgumentCountError::class);
```

Comment:
Since PHP 7.1 the warning has been replaced by an exception ( http://php.net/manual/en/migration71.incompatible.php#migration71.incompatible.too-few-arguments-exception )
I just replaced `error()->exists()` (https://github.com/M6Web/RedisMock/pull/61/files#diff-a0625becda4fa507dfe3e27141717e2eL136) by exception check